Packs/Automox/Integrations/Automox/Automox.yml

```diff
@@ -0,0 +1,1111 @@
+      type: String
+    - contextPath: Automox.CreatedGroups.notes
+      description: 'Notes defined for the group'
+      type: String
+    - contextPath: Automox.CreatedGroups.enable_os_auto_update
+      description: 'Enable operating system auto updates'
+      type: Boolean
+    - contextPath: Automox.CreatedGroups.server_count
+      description: 'Number of devices assigned to group'
+      type: Number
+    - contextPath: Automox.CreatedGroups.policies
+      description: 'List of policies assigned to group'
+      type: Number
+  - name: automox-group-update
+    description: "Update a group's information in Automox"
+    arguments:
+    - name: org_id
+      description: 'Organization identifier'
+      type: 'number'
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: group_id
+      description: 'Group identifier'
+      type: 'number'
+      isArray: false
+      required: true
+      default: false
+      secret: false
+    - name: name
+      description: 'Name of the group'
+      type: String
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: notes
+      description: 'Define notes for the group'
+      type: String
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: parent_server_group_id
+      description: 'Identifier of the parent group. Defaults to default group id if omitted.'
+      type: Number
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: policies
+      description: 'List of policy identifiers to assign to group'
+      type: Number
+      isArray: true
+      required: false
+      default: false
+      secret: false
+    - name: refresh_interval
+      description: 'Frequency of device refreshes in minutes'
+      type: Number
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: color
+      description: 'Automox console highlight color for the group. Value should be a valid Hex color code'
+      type: String
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    deprecated: false
+    execution: false
+  - name: automox-group-delete
+    description: "Delete a group from Automox"
+    arguments:
+    - name: org_id
+      description: 'Organization identifier'
+      type: Number
+      isArray: false
+      required: false
+      default: false
+      secret: false
+    - name: group_id
+      description: 'Group identifier'
+      type: Number
+      isArray: false
+      required: true
+      default: false
+      secret: false
+    deprecated: false
+    execution: false
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/python3:3.10.4.27798
+fromversion: 6.0.0
+tests:
+- Upload_Vulnerability_Report_to_Automox
```

Comment:
```suggestion
- No tests
```
I've checked again and this section needs to be empty (like you did in the beginning), sorry for the confusion

Reply:
No worries, thanks for adding the suggestion.